composer.json

```diff
@@ -18,7 +18,11 @@
             "email": "jacob@tighten.co"
         }
     ],
+    "suggest": {
+        "ext-inotify": "To use the watch option of the generate command"
+    },
     "require": {
+        "ext-json": "*",
```

Comment:
This was missing and is required to use json_encode / json_decode in php, even though this extension is usually installed by default in php